### DISCUSSION TITLE
광고차단해제 문구 제거

### DISCUSSION THREAD

**Discussion by taeklims** [List-KR/community#5] [General]
모바일로 애드가드에 list-kr 필터 적용하여 이용중에 있습니다.
최근 https://aagag.com/issue/ 커뮤니티 사이트 이용중 댓글보기란에 
![Screenshot_20230804_181553_Samsung Internet](https://github.com/List-KR/List-KR/assets/141406383/dc21ab44-825e-4c2a-94ad-e1e9066dcebb)
이미지처럼 광고 해제하라는 문구가 뜹니다. 바이패스하여 정상적으로 이용가능할까요?

**Comment by piquark6046** [comment]
네 가능합니다.

**Reply by piquark6046** [reply]
iOS 인가요? 아니면 Android 인가요?

**Reply by taeklims** [reply]
안드로이드 사용중입니다~~

**Reply by piquark6046** [reply]
그럼, 문제가 없겠군요.

**Reply by taeklims** [reply]
감사합니다~👍

**Reply by taeklims** [reply]
윈도우10 크롬 ublock에서도 adguard 필터와 list-kr 필터 적용시 동일 증상이 나타나네요...

**Reply by piquark6046** [reply]
uBO 지원 탭에서 Troubleshooting Information의 내용을 공유해주세요.
그 전에, 먼저 필터 리스트 업데이트는 하셨나요?

**Reply by taeklims** [reply]
넵 필터 업데이트 적용했습니다~
모바일은 잘 됩니다 😁

**Reply by piquark6046** [reply]
지금 문제가 해결된 건가요?

**Reply by taeklims** [reply]
PC 크롬에서는 안되네욥...

**Reply by seia-soto** [reply]
@piquark6046 이후 웹 사이트 내 팝업 보시게 되는 경우 라이브러리 차단 이후에 추가 확인 부탁드리겠습니다: `https://cdnjs.cloudflare.com/ajax/libs/blockadblock/3.2.1/blockadblock.min.js`

**Reply by piquark6046** [reply]
![image](https://github.com/List-KR/List-KR/assets/98787049/70b79516-ed48-4b49-ad13-fa0395deed51)

그 URL을 차단해도 똑같습니다.

@taeklims uBO 지원 탭에서 Troubleshooting Information의 내용을 공유해주세요.

**Reply by taeklims** [reply]
현재 필터 업데이트 후 정상적으로 잘 적용되고있습니다. 감사합니다🫡

**Comment by taeklims** [comment]
![Screenshot_20230808_235958_Samsung Internet](https://github.com/List-KR/List-KR/assets/141406383/a270fa08-5e6e-4fce-a81b-1194bac7e589)
막는 기능이 업데이트 되었나봅니다...
다시 광고회피가 안되게 뜨네요..

**Reply by piquark6046** [reply]
네. 업데이트했네요.

**Reply by piquark6046** [reply]
https://github.com/List-KR/List-KR/issues/769 에서 이어서 진행합니다.

**Reply by zlsmsdnjsqls** [reply]
또 업데이트 했나봐요...

**Reply by taeklims** [reply]
계속 막히는건가보네요...ㅠㅜ
